game_event/models.py

```diff
@@ -17,3 +20,27 @@ class GameEvent(models.Model):
         validators=[MinLengthValidator(4)],
         choices=BallGame.choices
     )
+
+    @staticmethod
+    def create(time, level_of_game, min_number_of_players, max_number_of_players, court, ball_game):
+        GameEvent.validate_game_event(court, ball_game, min_number_of_players, time)
+        game_event = GameEvent(time=time,
+                               level_of_game=level_of_game,
+                               min_number_of_players=min_number_of_players,
+                               max_number_of_players=max_number_of_players,
+                               court=court,
+                               ball_game=ball_game)
+        game_event.save()
+        return game_event
+
+    @staticmethod
+    def validate_game_event(court, ball_game, min_number_of_players, time):
```

Comment:
why dont we validate `max_number_of_players`?

Reply:
you are right, i can validate if max - min is > 0